ntp-daemon/src/system.rs

```diff
@@ -364,7 +371,35 @@ mod tests {
         let (msg_for_system_tx, msg_for_system_rx) = mpsc::channel::<MsgForSystem>(32);
         let global_system_snapshot = Arc::new(tokio::sync::RwLock::new(SystemSnapshot::default()));
 
-        let peers_rwlock = Arc::new(tokio::sync::RwLock::new(Peers::new(&test_peer_configs(4))));
+        // TODO: Fix this
```

Comment:
?

Reply:
This was a leftover from a refactoring step which I forgot to remove. It is gone now.